### PR TITLE
Changing luis sdk namespace and package version

### PR DESCRIPTION
**Pull request by mohabghanem**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/LUIS/Programmatic/readme.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ csharp:
   sync-methods: None
   license-header: MICROSOFT_MIT_NO_VERSION
   azure-arm: false
-  namespace: Microsoft.Azure.CognitiveServices.Language.LUIS.Programmatic
+  namespace: Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring
```
Comment (lmazuel) [inline specification/cognitiveservices/data-plane/LUIS/Programmatic/readme.md]: Seems the folder has to be renamed as well?

File: specification/cognitiveservices/data-plane/LUIS/Programmatic/stable/v2.0/LUIS-Programmatic.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "LUIS Programmatic API",
-    "version": "v2.0"
+    "version": "v2.0 preview"
```
Comment (lmazuel) [inline specification/cognitiveservices/data-plane/LUIS/Programmatic/stable/v2.0/LUIS-Programmatic.json]: Why is that?

**Comment by lmazuel** [comment]
Could you explain a little more the why of this PR? I miss context to judge if this is the right thing to do.
Adding @shahabhijeet if he has a comment, since it seems to be mostly C# changes

File: specification/cognitiveservices/data-plane/LUIS/Programmatic/stable/v2.0/LUIS-Programmatic.json (hunk 2 above)
Comment (bin2214) [reply specification/cognitiveservices/data-plane/LUIS/Programmatic/stable/v2.0/LUIS-Programmatic.json]: This package have never been published. When we publish the first version of a package, we would like to set it as preview first (like other Cognitive packages at https://www.nuget.org/packages?q=Microsoft.Azure.CognitiveServices. Then we will remove preview after it is stable later on.

File: specification/cognitiveservices/data-plane/LUIS/Programmatic/readme.md (hunk 1 above)
Comment (bin2214) [reply specification/cognitiveservices/data-plane/LUIS/Programmatic/readme.md]: +1 on this.

**Comment by bin2214** [comment]
The purpose of this PR is: 
1. change name space of SDK to match naming conversion of Cognitive Service API's. 
2. Generate first preview version of SDK 

Then we can publish it and update sample project to point to this SDK.

**Review by lmazuel — APPROVED** [review]

**Pull request merged**